config/index.js

```diff
@@ -0,0 +1,42 @@
+// Token duration config
+const durationRefreshToken = 129600; // 90 d‬
+const durationToken = 900; // 15 min
+
+// Error handler
+function centralErrorHandler(err, req, res, next) {
+  if (err.name === "UnauthorizedError") {
+    res.status(401).send(err);
+  } else if (err.message == "Not allowed by CORS") {
+    res.status(401).send("Not allowed from this origin.");
+  } else {
+    next(err);
+  }
+}
+
+// Whitelisted domains
+let whitelist = ["https://kaffeerudel.at", "http://localhost:3000"];
```

Comment:
constable goodie, what in the name of the queen are you doing?

Reply:
Oh thanks for commenting on this part of the code.

I did actually copy this code from another repository and didn't realise it is offensive.
The new official term for `whitelist` ist `allowlist`.

```suggestion
const allowlist = ["https://kaffeerudel.at", "http://localhost:3000"];
```